sdcm/db_log_reader.py

```diff
@@ -105,13 +106,16 @@ def _read_and_publish_events(self) -> None:
                         except Exception:  # pylint: disable=broad-except
                             pass
 
-                    if self._log_lines:
-                        line = line.strip()
-                        for pattern in self.EXCLUDE_FROM_LOGGING:
-                            if pattern in line:
-                                break
-                        else:
-                            LOGGER.debug(line)
+                    line = line.strip()
+
+                    for pattern in self.EXCLUDE_FROM_LOGGING:
+                        if pattern in line:
+                            should_exclude_line = True
+                            break
+                    else:
+                        should_exclude_line = True
```

Comment:
Typo here, it should be `False`.

Probably it is the reason for the great perf changes?

Reply:
changed, it's just if it would printed in sct.log, I don't think it has and real effect on the cpu usage